compiler/src/dotty/tools/dotc/semanticdb/Annotation.scala

```diff
@@ -0,0 +1,89 @@
+// Generated by the Scala Plugin for the Protocol Buffer Compiler.
+// Do not edit!
+//
+// Protofile syntax: PROTO3
```

Comment:
Maybe I should comment it is generated from `tanishiking/semanticdb-for-scala3` ?